docs/developer-guide/governance.md

```diff
@@ -34,7 +34,8 @@ Committers:
 * May submit small changes (documentation updates, changes to tests or code comments, configuration changes) without pull requests.
 * Must submit pull requests for any non-trivial changes.
 * Have their work reviewed by Reviewers before acceptance into the repository.
-* Can label issues as they are submitted (but must not add the "accepted" label).
+* May label issues as they are submitted ("accepted" label should only be added for bugs, and only if the committer verified the bug as valid).
```

Comment:
Yes, that's what this says. You, the committer, can verify the bug and mark as accepted.